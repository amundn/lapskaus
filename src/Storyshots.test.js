import initStoryshots from '@storybook/addon-storyshots'

//jest.mock('fetch');

initStoryshots()
