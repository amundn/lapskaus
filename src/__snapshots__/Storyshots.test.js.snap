// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Artshjul mockup 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <img
          src="mockup.png"
        />
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Kart mockup 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <img
          src="mockup.png"
        />
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Statistikk mockup 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <img
          src="mockup.png"
        />
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;
